sql/src/test/java/org/apache/druid/sql/avatica/DruidAvaticaHandlerTest.java

```diff
@@ -918,6 +918,107 @@ public Frame fetch(
     );
   }
 
+
+  @Test
+  public void testMinRowsPerFrame() throws Exception
```

Comment:
Trying to fully understand this test. Is this testing that if a client supplied rows per frame is less than the configured minimum rows per frame, we stick with the minimum in the config?

Reply:
Yep, the [statement sets a fetch size of 2](https://github.com/apache/druid/pull/10880/files#diff-5430fa325aa463d4b6c34cad0fa3816e30e189ad2e3d7dbf56567c60e9c6c785R1004), but the min is 1000, so prior to this patch this test would fail because it would take multiple frames (and it also explicitly checks that fetch is called with the adjusted minimum value here https://github.com/apache/druid/pull/10880/files#diff-5430fa325aa463d4b6c34cad0fa3816e30e189ad2e3d7dbf56567c60e9c6c785R978)

Reply:
thanks for the clarification